app/src/main/res/layout/activity_booking_detail.xml

```diff
@@ -394,6 +299,177 @@
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:layout_alignParentBottom="true"
+                                    android:alpha="@dimen/float_0.6"
+                                    android:background="@color/color_transparent"
+                                    android:gravity="center"
+                                    android:text="Left"
+                                    android:textColor="@color/color_white"/>
+                            </RelativeLayout>
+                        </android.support.v7.widget.CardView>
+
+                        <android.support.v7.widget.CardView
+                            android:layout_width="0dp"
+                            android:layout_height="wrap_content"
+                            android:layout_marginRight="@dimen/dp_3"
+                            android:layout_weight="1"
+                            car_view:cardElevation="@dimen/dp1">
+
+                            <RelativeLayout
+                                android:layout_width="match_parent"
+                                android:layout_height="@dimen/dp_80"
+                                android:onClick="@{()->viewModel.pickImage(SideCapture.SIDE_RIGHT)}">
+
+                                <ImageView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="match_parent"
+                                    android:scaleType="centerCrop"
+                                    bind:imageUrl="@{viewModel.pathImageSideRight}"
+                                    tool:ignore="ContentDescription"/>
+
+                                <ImageView
+                                    android:layout_width="wrap_content"
+                                    android:layout_height="wrap_content"
+                                    android:layout_centerInParent="true"
+                                    android:alpha="@dimen/float_0.6"
+                                    android:visibility="@{viewModel.pathImageSideRight == null ? View.VISIBLE : View.GONE}"
+                                    bind:srcCompat="@drawable/ic_camera_24dp"
+                                    tool:ignore="ContentDescription"/>
+
+                                <TextView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:layout_alignParentBottom="true"
+                                    android:alpha="@dimen/float_0.6"
+                                    android:background="@color/color_transparent"
+                                    android:gravity="center"
+                                    android:padding="@dimen/dp_1"
+                                    android:text="Right"
```

Comment:
WARNING: [HardcodedText] [I18N] Hardcoded string "Right", should use `@string` resource
 Hardcoding text attributes directly in layout files is bad for several reasons:

* When creating configuration variations (for example for landscape or portrait)you have to repeat the actual text (and keep it up to date when making changes)

* The application cannot be translated to other languages by just adding new translations for existing string resources.

There are quickfixes to automatically extract this hardcoded string into a resource lookup.